rhui_provision/tasks/upgrade_ssl_configuration.yml

```diff
@@ -0,0 +1,24 @@
+- name: Backup old ssl configuration
+  command: mv /etc/httpd/conf.d/ssl.conf /etc/httpd/conf.d/ssl.conf.bak
+
+- name: Copy and install new ssl configuration
+  copy: 
+    src: "files/ssl.conf" 
+    dest: "/etc/httpd/conf.d/ssl.conf" 
+    owner: root
+    group: root
+    mode: 0644
+
+
+- name: Generate DH params for cert
+  command: openssl dhparam -out /etc/pki/cds/dh2048.pem 2048
+
+- name: Backup old ssl cert
+  command: cp /etc/pki/cds/cds-ssl-cert.crt /etc/pki/cds/cds-ssl-cert.crt.bak
+
+- name: Concate DH params to SSL cert
+  command: cat /etc/pki/cds/dh2048.pem >> /etc/pki/cds/cds-ssl-cert.crt
```

Comment:
Do you any thoughts of how we could make this idempotent, so we could re-run this ansible code on the same server multiple times without appending the 2048 key to the cert multiple times?